apps/journeys-admin/src/components/JourneyList/JourneyList.stories.tsx

```diff
@@ -2,16 +2,23 @@ import { Story, Meta } from '@storybook/react'
 
 import { journeysAdminConfig } from '../../libs/storybook'
 import JourneyList from './JourneyList'
-import { defaultJourney } from './journeyListData'
+import {
+  defaultJourney,
+  oldJourney,
+  descriptiveJourney
+} from './journeyListData'
 
 const TestStory = {
   ...journeysAdminConfig,
   component: JourneyList,
   title: 'Journeys-Admin/JourneyList'
 }
 
-const Template: Story = () => <JourneyList journeys={[defaultJourney]} />
+const Template: Story = ({ ...args }) => <JourneyList journeys={[]} {...args} />
```

Comment:
```suggestion
const Template: Story<JourneyListProps> = ({ ...args }) => <JourneyList {...args} />
```

And export JourneyListProps like how we did for the others

Reply:
If you set the type of Story as <JourneyListProps> this avoids you having to specify a placeholder `journeys={[]}` for every prop on this component (good practice for when components have multiple props / since we're using typescript)